public/CSS/style.css

```
@@ -0,0 +1,161 @@
+/** reset **/
+* {
+    margin: 0;
+    padding: 0;
+  }
+
+  a {
+    text-decoration: none;
+    color: inherit;
+  }
+  
+  /* True Styling */
+  
+  header {
+    background-image: url(https://actualitte.com/media/cache/width_944_webp/uploads/images/un-nouveau-projet-d-hayao-miyazaki-inspire-d-un-classique-de-la-litterature-japonaise-6950fb53c799.jpg);
+    background-size: cover; 
+    background-repeat: no-repeat;
+    display: flex;
+    flex-direction: column;
+  }
+  
+  header h1 {
+    margin: 0;
+    font-family: fantasy;
+    font-size: 5rem;
+    padding: 10px;
+    text-align: center;
+    color: white!important;
+
+    text-shadow: 0px 0px 10px rgba(0,0,0,0.5), 0px 0px 10px rgba(0,0,0,0.5), 0px 0px 10px rgba(0,0,0,0.5);
+  }
+  
+  header nav{
+    font-family: 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
+    flex-direction: column;
+  }
+  
+  header nav ul {
+    background: rgba(255, 255, 255, 0.486);
+    color: gb(243, 205, 18);
+    display: flex;
+    justify-content: center;
+    list-style-type: none;
+    height: 50px;
+  }
+  
+  header nav ul li {
+    text-align:center;
+    padding: 15px;
+    transition: all ease .5s;
+    font-size: 1rem;
+  }
+  
+  header nav ul li:hover {
+    font-size: 1.2rem;
+    text-shadow: 0px 10px 4px rgba(0,0,0,0.5);
+  }
+  
+  h3 {
+    text-align: center;
+    line-height: 3rem;
+  }
+  
+  footer {
+    margin-top: 2.5%;
+    background-color: rgba(255, 255, 255, 0.493);
+  }
+
+.filmList {
+    color: white;
+    background-color: rgba(2, 1, 100, 0.8);
+    padding: 1.5em;
+    border: solid white 0.1em;
+    border-radius: 5%;
+
+}
+
+p {
+    margin-top: 5rem;
+    margin-left: 40rem;
+    margin-right: 40rem;
+    text-align: justify;
+    font-weight: bold;
+    background-color: rgba(255, 255, 255, 0.493);
+}
+
+
+
+.filmForm {
+    width: 58vh;
+    height: 30vh;
+    background-size: cover; 
+    background-repeat: no-repeat;
+}
+
+.filmTitle {
```

```suggestion
.film-title {
```

generalement, c'est du `kebab-case` sur le CSS, mais c'est du détail ici.